app/components/UI/Navbar/index.js

```diff
@@ -682,17 +686,61 @@ export function getOnboardingNavbarOptions(
  *
  * @returns {Object} - Corresponding navbar options containing headerTitle
  */
-export function getTransparentOnboardingNavbarOptions(themeColors) {
+export function getTransparentOnboardingNavbarOptions(
+  themeColors,
+  showLogo = true,
+  color,
+  darkColor = false,
+) {
   const innerStyles = StyleSheet.create({
     headerStyle: {
-      backgroundColor: themeColors.background.default,
+      backgroundColor: color || themeColors.background.default,
```

Comment:
We use a lot of the name color in the params and we only use it for the backgroundColor, would be more accurate to rename it?

Reply:
replaced with headerBgColor
https://github.com/Web3Auth/metamask-mobile/pull/53/commits/e0de428abfefd66b113548edd94235190d134566